src/Features/LanguageServer/ProtocolUnitTests/References/FindAllReferencesHandlerTests.cs

```diff
@@ -37,34 +35,8 @@ void M2()
 }";
             using var workspace = CreateTestWorkspace(markup, out var locations);
 
-            var results = await RunFindAllReferencesAsync(workspace.CurrentSolution, locations["caret"].First(), true);
-            AssertLocationsEqual(locations["reference"], results);
-        }
-
-        [WpfFact]
-        public async Task TestFindAllReferencesAsync_DoNotIncludeDeclarations()
-        {
-            var markup =
-@"class A
-{
-    public int someInt = 1;
-    void M()
-    {
-        var i = {|reference:someInt|} + 1;
-    }
-}
-class B
-{
-    int someInt = A.{|reference:someInt|} + 1;
-    void M2()
-    {
-        var j = someInt + A.{|caret:|}{|reference:someInt|};
-    }
-}";
```

Comment:
why was this test removed?

Reply:
In Roslyn's local FAR experience, I didn't see an option to not include definitions. (I might be totally off base here, feel free to correct me if so.) Additionally, in the new ReferenceItem class, I believe it works by associating every reference with a definition, so I'm not sure if it's an option to leave out the definition (at least on our end).

Reply:
in general, try not to remove tests.  instead, just update the test to the new 'expected' behavior.

Reply:
Gotcha. For this test in particular, I believe the expected behavior is identical to the test right above it, so I don't know if there would be value in keeping it around.

Reply:
good to know. that's an ok reason to remove.  if the test is identical though, please comment as such in the PR so that's known :)